ICRCs/ICRC-122/ICRC-122.md

```diff
@@ -0,0 +1,150 @@
+# ICRC-122: Token Burning & Minting (Consolidated)
```

Comment:
why is "Consolidated" here explicitly mentioned? can you provide me some context?

I am also wondering if ICRC-122 is meant to be an extension to ICRC-1. if it is not an extension:
- CAN it be implemented among with ICRC-1?
  - is this what we plan to do in the reference implementation?
- MUST it be implemented independent of ICRC-1?

Reply:
remnant of a merge